app/models/service.rb

```diff
@@ -1,6 +1,10 @@
+require 'uri/ssh_git'
```

Comment:
Is this needed? I think rails should require this automatically

Reply:
i have removed this